device_calendar/example/lib/presentation/pages/calendar_event.dart

```diff
@@ -14,11 +15,14 @@ class CalendarEventPage extends StatefulWidget {
   final Calendar _calendar;
   final Event _event;
 
-  CalendarEventPage(this._calendar, [this._event]);
+  final VoidCallback _onLoadingStarted;
+  final Function(bool) _onDeleteFinished;
+
+  CalendarEventPage(this._calendar, [this._event, this._onLoadingStarted, this._onDeleteFinished]);
```

Comment:
why are we passing callbacks into the page constructor - this doesn't seem like a good idea.

Reply:
Removed it and now passes a widget instead